tests/integration/ha/helpers.py

```diff
@@ -504,24 +514,62 @@ async def start_and_check_continuous_writes(ops_test: OpsTest, unit_ip: str, app
     )
     writer = ContinuousWrites(ops_test, app, initial_count=initial_count)
     await writer.start()
-    time.sleep(10)
-    result = await writer.stop()
-    return result.count > initial_count
+    time.sleep(60)
+    # Ensure we have writes happening and the index is consistent at the end
+    await assert_continuous_writes_increasing(writer)
+    await assert_continuous_writes_consistency(ops_test, writer, app)
```

Comment:
just FYI if this check fails then writes won't be cleared. (i.e. if a customer runs this test on their deployed cluster then they will have a bunch of "junk/test" writes. Adding a fixure to tests that clears writes after tests can be useful